pxr/imaging/hd/material.h

```diff
@@ -26,9 +26,10 @@ class HdMaterial : public HdSprim
     enum DirtyBits : HdDirtyBits {
         Clean                 = 0,
         // XXX: Got to skip varying and force sync bits for now
+        DirtyInit             = 1 << 1, // Sprim created, but never synced
```

Comment:
We need to differentiate between materials which are dirty because they've just been created but never synced, and those that were dirtied by editing. When we use the optimization, we cache the state as of the material's creation, so if it's edited after the creation and before `Sync()`, that state will become stale. This was caught by animated material unit tests. I'll comment below how we handle the stale state.